docs/about/whats_new.rst

```diff
@@ -5,10 +5,7 @@
 What's New
 **********
 
-Next Version
-============
-
-v1.9.4 (?? May 2025)
+v1.9.4 (19 May 2025)
```

Comment:
20 May - sorry didn't see this last night.

Reply:
The #1780 Dask bug fix is worth mentioning - we forgot to whats_new it at the time.

Reply:
Thanks, I'll tidy things up

Reply:
It's still the 19th on the American continents...

(I will change it)